src/cpp/rtps/messages/MessageReceiver.cpp

```diff
@@ -1021,17 +1054,16 @@ bool MessageReceiver::proc_Submsg_HeartbeatFrag(CDRMessage_t*msg, SubmessageHead
 
     std::lock_guard<std::mutex> guard(mtx);
     //Look for the correct reader and writers:
+    /* XXX TODO
```

Comment:
As no processing is done, we can move the beginning of the comment block before the lock guard.